### ISSUE TITLE
Behavior of `emp restart` is confusing

### ISSUE THREAD

**Issue by ejholmes** [remind101/empire#697]
Currently `emp restart` has the following behavior:

* `emp restart` will do a rolling restart of all processes. The implementation is weird since we perform a new release, when we should have a `Restart` method on the Scheduler interface.
* `emp restart <ptype>` returns an error `InvalidParameterException: taskId shorter than 36`.
* `emp restart <ecs taskid>` kills the ECS task.

With that said, the behavior we probably want is:

* `emp restart` to behave as above, but fix implementation details.
* `emp restart <ptype>` should do a rolling restart of the ECS service. Implementation should be pretty easy if we fix above.
* `emp restart <ecs taskid>` I'm not sure we actually want to keep this. It's almost never used and can be potentially dangerous since it doesn't do a rolling restart.

/cc @phobologic since you were talking about fixing this.

**Comment by phobologic** [comment]
Yeah, that's actually where I was thinking we'd go with this.  I had the same concerns about the taskID method, then got pulled off on other stuff.
